From 7d287d826c9755381df33b219bc46c930379e9b6 Mon Sep 17 00:00:00 2001
From: Mikhail Kasimov <mikhail.kasimov@gmail.com>
Date: Fri, 11 May 2018 02:17:27 +0300
Subject: [PATCH] man: systemd-journal-upload: rsa 2048

rsa:2048 is currently in use as minimal recommended key length on cert/key generating. Specifying rsa:1024 is not recommended for security reasons.
(cherry picked from commit 562b65ca38cfe79bce1d33bec363fe72f903a8b6)
---
 man/systemd-journal-upload.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/systemd-journal-upload.xml b/man/systemd-journal-upload.xml
index a43062a6d5..fecffd5d9b 100644
--- a/man/systemd-journal-upload.xml
+++ b/man/systemd-journal-upload.xml
@@ -268,10 +268,10 @@ echo 0001 &gt;serial
 SERVER=server
 CLIENT=client
 
-openssl req -newkey rsa:1024 -nodes -out $SERVER.csr -keyout $SERVER.key -subj "/CN=$SERVER/"
+openssl req -newkey rsa:2048 -nodes -out $SERVER.csr -keyout $SERVER.key -subj "/CN=$SERVER/"
 openssl ca -batch -config ca.conf -notext -in $SERVER.csr -out $SERVER.pem
 
-openssl req -newkey rsa:1024 -nodes -out $CLIENT.csr -keyout $CLIENT.key -subj "/CN=$CLIENT/"
+openssl req -newkey rsa:2048 -nodes -out $CLIENT.csr -keyout $CLIENT.key -subj "/CN=$CLIENT/"
 openssl ca -batch -config ca.conf -notext -in $CLIENT.csr -out $CLIENT.pem
 </programlisting>
 
-- 
2.17.0

